src/libraries/System.IO.FileSystem.Watcher/tests/FileSystemWatcher.unit.cs

```diff
@@ -1122,30 +1100,29 @@ public void FileSystemWatcher_CreateManyConcurrentWatches()
         {
             int maxUserWatches = int.Parse(File.ReadAllText("/proc/sys/fs/inotify/max_user_watches"));
 
-            using (var dir = new TempDirectory(GetTestFilePath()))
-            using (var watcher = new FileSystemWatcher(dir.Path) { IncludeSubdirectories = true, NotifyFilter = NotifyFilters.FileName })
+            using (var watcher = new FileSystemWatcher(TestDirectory) { IncludeSubdirectories = true, NotifyFilter = NotifyFilters.FileName })
             {
                 Action action = () =>
                 {
                     // Create enough directories to exceed the number of allowed watches
                     for (int i = 0; i <= maxUserWatches; i++)
                     {
-                        Directory.CreateDirectory(Path.Combine(dir.Path, i.ToString()));
+                        Directory.CreateDirectory(Path.Combine(TestDirectory, i.ToString()));
```

Comment:
```suggestion
                        CreateDirectory(TestDirectory, i.ToString());
```

Reply:
@danmoseley  Do you mean `CreateTestDirectory` here? Also, do we need to add it here? We are not using any return values and therefore I think it would be redundant to use them here.